### PR TITLE
feat(JumpLinks): add tokens

### LINKED ISSUE #6189: Tokens: Jump links

### PR DESCRIPTION
**Pull request by srambach**
Fixes #6189

### CODE REVIEW

**Review by andrew-ronaldson — APPROVED** [review]
Design looks good. Thanks!

File: src/patternfly/components/JumpLinks/jump-links.scss [hunk 1]
```diff
@@ -1,79 +1,82 @@
...
+  --#{$jump-links}--m-vertical__item--m-current__link--before--BorderLeftWidth: var(--pf-t--global--border--width--extra-strong);
 
   // text
-  --#{$jump-links}__link-text--Color: var(--#{$pf-global}--Color--200);
-  --#{$jump-links}__link--hover__link-text--Color: var(--#{$pf-global}--Color--100);
-  --#{$jump-links}__link--focus__link-text--Color: var(--#{$pf-global}--Color--100);
-  --#{$jump-links}__item--m-current__link-text--Color: var(--#{$pf-global}--Color--100);
+  --#{$jump-links}__link-text--Color: var(--pf-t--global--text--color--subtle);
+  --#{$jump-links}__link--hover__link-text--Color: var(--pf-t--global--text--color--subtle);
+  --#{$jump-links}__link--focus__link-text--Color: var(--pf-t--global--text--color--subtle);
```
Comment (thatblindgeye) [inline src/patternfly/components/JumpLinks/jump-links.scss:52]: As a follow up we should remove the focus var and combine it and hover blocks

**Comment by lboehling** [comment]
lgtm too!

**Review by kmcfaul — APPROVED** [review]

**Review by thatblindgeye — COMMENTED** [review]
Nit below that can be a followup. Also currently the designs show a help icon button next to the label text. Right now it doesn't look like there's styling to take that into account?

**Comment by lboehling** [comment]
lgtm! removed the disabled variant from figma, don't think we need it for jumplinks, but can add in later if it's requested.

**Review by thatblindgeye — APPROVED** [review]

**Pull request merged**